### ISSUE TITLE
Docksal seems to cause CPU spikes

### ISSUE THREAD

**Issue by wfraga** [docksal/docksal#1037]
**Description**

I'm having problems of performance when running docksal locally. Basically I have many CPU spikes when running docksal that produces a great slowness in the system. 

I saw in my local system log some entries that can explain the reason of this problem.

Someone can help me with this issue? 

**System log entry**

<details>
  <summary>syslog</summary>

```
Apr 18 14:53:40 wfraga-VPCEG15FB dockerd[2604]: time="2019-04-18T14:53:40.297813947-03:00" level=warning msg="Health check for container 78ab82840c106daa302e398e706487fd509c76e173a8ebe1c9abb0782fb473f6 error: context deadline exceeded"

Apr 18 14:53:40 wfraga-VPCEG15FB dockerd[2604]: time="2019-04-18T14:53:40.604102042-03:00" level=warning msg="Ignoring Exit Event, no such exec command found" container=12f3af1ad346d6be8feecd58e4ce3263f906213267a947fb357f768e11d8d309 exec-id=23aac464f647d05f8770a2076afbad83fb7ee7516978290f90f4a2579e242282 exec-pid=6743

```

</details>

<br>

**Output of `fin sysinfo`:**

<details>
  <summary>fin sysinfo</summary>

  ```
███  OS
Linux Linux Mint 18.3
Linux wfraga-VPCEG15FB 4.15.0-47-generic #50~16.04.1-Ubuntu SMP Fri Mar 15 16:06:21 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

███  ENVIRONMENT
MODE : Linux Kernel
DOCKER_HOST : 

███  FIN
fin version: 1.85.0

███  DOCKER COMPOSE
EXPECTED VERSION: 1.23.2
docker-compose version 1.23.2, build 1110ad01
docker-py version: 3.6.0
CPython version: 3.6.7
OpenSSL version: OpenSSL 1.1.0f  25 May 2017

███  DOCKER
EXPECTED VERSION: 18.09.2

Client:
 Version:           18.09.5
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        e8ff056dbc
 Built:             Thu Apr 11 04:44:28 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.5
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.8
  Git commit:       e8ff056
  Built:            Thu Apr 11 04:10:53 2019
  OS/Arch:          linux/amd64
  Experimental:     false

███  DOCKSAL: DNS
Successfully requested http://dns-test.docksal

███  DOCKER: RUNNING CONTAINERS
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                       PORTS                                                    NAMES
e0eadd73b84b        docksal/ssh-agent:1.2       "docker-entrypoint.s…"   About an hour ago   Up About an hour (healthy)                                                            docksal-ssh-agent
78ab82840c10        docksal/dns:1.1             "docker-entrypoint.s…"   About an hour ago   Up About an hour (healthy)   192.168.64.100:53->53/udp                                docksal-dns
fdc94a4e90bd        docksal/vhost-proxy:1.5     "docker-entrypoint.s…"   About an hour ago   Up About an hour (healthy)   192.168.64.100:80->80/tcp, 192.168.64.100:443->443/tcp   docksal-vhost-proxy
fdfc160701fb        docksal/web:2.1-apache2.2   "httpd-foreground"       8 days ago          Up About an hour             80/tcp, 443/tcp                                          flyontario_web_1
12f3af1ad346        docksal/cli:1.3-php5        "/opt/startup.sh sup…"   8 days ago          Up About an hour (healthy)   22/tcp, 9000/tcp                                         flyontario_cli_1
2915986ade7c        mailhog/mailhog             "MailHog"                8 days ago          Up About an hour             80/tcp, 1025/tcp, 8025/tcp                               flyontario_mail_1
8754424be103        docksal/db:1.1-mysql-5.5    "/entrypoint.sh mysq…"   8 days ago          Up About an hour             0.0.0.0:32769->3306/tcp                                  flyontario_db_1

███  DOCKER: NETWORKS
NETWORK ID          NAME                 DRIVER              SCOPE
b0f75ad1d62d        _default             bridge              local
f7cc10ec2fa8        bridge               bridge              local
9b1c42f043cb        flyontario_default   bridge              local
b56bce61e09b        host                 host                local
24ece9ea6cc5        none                 null                local

███  VIRTUALBOX
EXPECTED VERSION: 5.2.26
5.1.38_Ubuntur122592

███  HDD Usage
Filesystem                 Size  Used Avail Use% Mounted on
udev                       5,9G     0  5,9G   0% /dev
tmpfs                      1,2G  9,8M  1,2G   1% /run
/dev/mapper/mint--vg-root  909G  421G  442G  49% /
tmpfs                      5,9G  258M  5,6G   5% /dev/shm
tmpfs                      5,0M  8,0K  5,0M   1% /run/lock
tmpfs                      5,9G     0  5,9G   0% /sys/fs/cgroup
/dev/loop4                 264M  264M     0 100% /snap/phpstorm/90
/dev/loop5                  92M   92M     0 100% /snap/core/6531
/dev/loop2                 261M  261M     0 100% /snap/phpstorm/87
/dev/loop0                 261M  261M     0 100% /snap/phpstorm/81
/dev/loop3                  90M   90M     0 100% /snap/core/6673
/dev/loop1                  91M   91M     0 100% /snap/core/6405
/dev/sda1                  472M  134M  315M  30% /boot
cgmfs                      100K     0  100K   0% /run/cgmanager/fs
tmpfs                      1,2G   60K  1,2G   1% /run/user/1000
/home/wfraga/.Private      909G  421G  442G  49% /home/wfraga

  ```

</details>

**Sysinfo**

<details>
  <summary>fin config output</summary>

```
---------------------
COMPOSE_PROJECT_NAME_SAFE: flyontario
COMPOSE_FILE:
/home/wfraga/.docksal/stacks/volumes-bind.yml
/home/wfraga/.docksal/stacks/stack-default.yml
/var/www/mmda/flyontario/.docksal/docksal.yml
ENV_FILE:
/var/www/mmda/flyontario/.docksal/docksal.env

PROJECT_ROOT: /var/www/mmda/flyontario
DOCROOT: docroot
VIRTUAL_HOST: flyontario.docksal
VIRTUAL_HOST_ALIASES: *.flyontario.docksal
IP: 192.168.64.100
MYSQL: 192.168.64.100:32769

Docker Compose configuration
---------------------
services:
  cli:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      BLACKFIRE_CLIENT_ID: null
      BLACKFIRE_CLIENT_TOKEN: null
      DOCROOT: docroot
      DRUSH_ALLOW_XDEBUG: '0'
      GIT_USER_EMAIL: **********@gmail.com
      GIT_USER_NAME: wfraga
      HOST_GID: '1000'
      HOST_UID: '1000'
      PHP_IDE_CONFIG: serverName=flyontario.docksal
      SECRET_ACAPI_EMAIL: null
      SECRET_ACAPI_KEY: null
      SECRET_PLATFORMSH_CLI_TOKEN: null
      SECRET_SSH_PRIVATE_KEY: null
      SECRET_TERMINUS_TOKEN: null
      VIRTUAL_HOST: flyontario.docksal
      XDEBUG_CONFIG: remote_connect_back=0 remote_host=192.168.64.1
      XDEBUG_ENABLED: '0'
    hostname: cli
    image: docksal/cli:1.3-php5
    volumes:
    - docksal_ssh_agent:/.ssh-agent:ro
    - cli_home:/home/docker:rw
    - project_root:/var/www:rw,nocopy
  db:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      MYSQL_ALLOW_EMPTY_PASSWORD: null
      MYSQL_DATABASE: default
      MYSQL_INITDB_SKIP_TZINFO: null
      MYSQL_ONETIME_PASSWORD: null
      MYSQL_PASSWORD: user
      MYSQL_RANDOM_ROOT_PASSWORD: null
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
    hostname: db
    image: docksal/db:1.1-mysql-5.5
    ports:
    - 3306/tcp
    volumes:
    - db_data:/var/lib/mysql:rw
    - project_root:/var/www:ro,nocopy
  mail:
    environment:
      MH_API_BIND_ADDR: 0.0.0.0:80
      MH_UI_BIND_ADDR: 0.0.0.0:80
    expose:
    - '80'
    hostname: mail
    image: mailhog/mailhog
    labels:
      io.docksal.virtual-host: webmail.flyontario.docksal
    user: root
  web:
    depends_on:
      cli:
        condition: service_started
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      APACHE_BASIC_AUTH_PASS: null
      APACHE_BASIC_AUTH_USER: null
      APACHE_DOCUMENTROOT: /var/www/docroot
      APACHE_FCGI_HOST_PORT: cli:9000
    hostname: web
    image: docksal/web:2.1-apache2.2
    labels:
      io.docksal.cert-name: none
      io.docksal.permanent: "false"
      io.docksal.project-root: /var/www/mmda/flyontario
      io.docksal.virtual-host: flyontario.docksal,*.flyontario.docksal,flyontario.docksal.*
    volumes:
    - project_root:/var/www:ro,nocopy
version: '2.1'
volumes:
  cli_home: {}
  db_data: {}
  docksal_ssh_agent:
    external: true
    name: docksal_ssh_agent
  project_root:
    driver: local
    driver_opts:
      device: /var/www/mmda/flyontario
      o: bind
      type: none

---------------------


```

</details>

**Comment by achekulaev** [comment]
1. How do you know it's Docksal causing spikes?
2. Could you include output of `fin sysinfo --all`

**Comment by wfraga** [comment]
hey @achekulaev ,

1. because it just happens when I start docksal (fin up) and because I compared these logs entry with my CPU reports:

![Screenshot from 2019-04-18 15-00-37](https://user-images.githubusercontent.com/11522001/56384622-425f4d80-61f3-11e9-9169-d5a9557c7d7f.png)

2. Right now, I am removing Docker and docksal completely to see if it is related to some old container or config. I hope this solves the problem, but if not, I can add what you asked for.

**Comment by achekulaev** [comment]
Whoa! That is a pretty intense CPU load. Nothing even remotely like this should happen/happened. Especially on Linux where FS is native so there is nothing to spin wheels for.

**Comment by wfraga** [comment]
hey @achekulaev ,

I've reinstalled docker, so I'm sending again my log entries and system infos:

**System log entry**

<details>
  <summary>syslog</summary>

```
Apr 26 14:31:33 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:33.856512585-03:00" level=error msg="stream copy error: reading from a closed fifo"
Apr 26 14:31:33 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:33.856512522-03:00" level=error msg="stream copy error: reading from a closed fifo"
Apr 26 14:31:34 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:34.294249372-03:00" level=error msg="stream copy error: reading from a closed fifo"
Apr 26 14:31:34 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:34.294403967-03:00" level=error msg="stream copy error: reading from a closed fifo"
Apr 26 14:31:34 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:34.294449671-03:00" level=error msg="stream copy error: reading from a closed fifo"
Apr 26 14:31:34 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:34.294733385-03:00" level=error msg="stream copy error: reading from a closed fifo"
Apr 26 14:31:34 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:34.456673449-03:00" level=warning msg="Health check for container 364acab01d5e11e8deade8a50573af5e63ad943a65ddfc9de72ffe0937fe9ecb error: context deadline exceeded: unknown"
Apr 26 14:31:34 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:34.530645432-03:00" level=warning msg="Health check for container 7e9344a2157f0ebc380859d1a993e7aa6879df3d34d5126e008311cb9d9ad23c error: context deadline exceeded: unknown"
Apr 26 14:31:34 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:31:34.549718619-03:00" level=warning msg="Health check for container 458dfeb533db62d5ece55547e9bcdfa39c743e9a08b34772b88401050092800a error: context deadline exceeded: unknown"
Apr 26 14:32:18 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:32:18.732532572-03:00" level=warning msg="Health check for container a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 error: context deadline exceeded"
Apr 26 14:32:18 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:32:18.857536509-03:00" level=warning msg="Ignoring Exit Event, no such exec command found" container=a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 exec-id=d29d2aac3cbf8f1bdb9e7461be865bca5e3192d2b681e82594a863df6fb725e0 exec-pid=25236
Apr 26 14:34:16 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:34:16.672130539-03:00" level=warning msg="Health check for container a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 error: context deadline exceeded"
Apr 26 14:34:16 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:34:16.795938333-03:00" level=warning msg="Ignoring Exit Event, no such exec command found" container=a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 exec-id=72fa33c7112eab1d62978442536c4f9aeefc9162385049b0826d79bf5169f39a exec-pid=29384
Apr 26 14:35:07 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:35:07.066497268-03:00" level=warning msg="Health check for container a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 error: context deadline exceeded"
Apr 26 14:35:07 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:35:07.090222804-03:00" level=warning msg="Ignoring Exit Event, no such exec command found" container=a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 exec-id=45ba1bb70aed8aab3b191b01563796bd63bfe88addc97e6272706acd6878d87f exec-pid=31175
Apr 26 14:38:25 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:38:25.154207719-03:00" level=warning msg="Health check for container a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 error: context deadline exceeded"
Apr 26 14:38:25 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:38:25.296289020-03:00" level=warning msg="Ignoring Exit Event, no such exec command found" container=a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 exec-id=39a7e8fef76d492bb10a6756dbb293cc7c64bbb994c2eaff2b63fb10f904b930 exec-pid=6408
Apr 26 14:38:33 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:38:33.771690783-03:00" level=warning msg="Health check for container a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 error: context deadline exceeded"
Apr 26 14:38:33 wfraga-VPCEG15FB dockerd[2434]: time="2019-04-26T14:38:33.781139329-03:00" level=warning msg="Ignoring Exit Event, no such exec command found" container=a22f8d0b7d949f22ac5b27a107de80917af997c276ca08f0b1e5f0aa3eba2cb4 exec-id=595684b52d97a988867195682178ff878d450e8530fc19d424bcd6e282613b8e exec-pid=6628


```

</details>

<br>

**Output of `fin sysinfo --all`:**

<details>
  <summary>fin sysinfo --all</summary>

  ```
███  OS
Linux Linux Mint 18.3
Linux wfraga-VPCEG15FB 4.15.0-48-generic #51~16.04.1-Ubuntu SMP Fri Apr 5 12:01:12 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

███  ENVIRONMENT
MODE : Linux Kernel
DOCKER_HOST : 

███  FIN
fin version: 1.85.1

███  DOCKER COMPOSE
EXPECTED VERSION: 1.23.2
docker-compose version 1.23.2, build 1110ad01
docker-py version: 3.6.0
CPython version: 3.6.7
OpenSSL version: OpenSSL 1.1.0f  25 May 2017

███  DOCKER
EXPECTED VERSION: 18.09.2

Client:
 Version:           18.09.5
 API version:       1.39
 Go version:        go1.10.8
 Git commit:        e8ff056dbc
 Built:             Thu Apr 11 04:44:28 2019
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.5
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.8
  Git commit:       e8ff056
  Built:            Thu Apr 11 04:10:53 2019
  OS/Arch:          linux/amd64
  Experimental:     false

███  DOCKER INFO
Containers: 21
 Running: 7
 Paused: 0
 Stopped: 14
Images: 16
Server Version: 18.09.5
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: bb71b10fd8f58240ca47fbb579b9d1028eea7c84
runc version: 2b18fe1d885ee5083ef9f0838fee39b62d653e30
init version: fec3683
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.15.0-48-generic
Operating System: Linux Mint 18.3
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 11.68GiB
Name: wfraga-VPCEG15FB
ID: E5KX:FCHY:L4HC:5TJT:JN5J:2XY2:PVLM:2OFX:NYOF:DYRP:6KBE:NBSY
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Product License: Community Engine

WARNING: No swap limit support

███  DOCKSAL: PROJECTS
project             STATUS              virtual host                                                   project root
flyontario          Up 19 minutes       flyontario.docksal,*.flyontario.docksal,flyontario.docksal.*   /var/www/mmda/flyontario

███  DOCKSAL: VIRTUAL HOSTS
*.flyontario.docksal
flyontario.docksal.*
flyontario.docksal
webmail.flyontario.docksal

███  DOCKSAL: DNS
Successfully requested http://dns-test.docksal

███  DOCKER: RUNNING CONTAINERS
CONTAINER ID        IMAGE                       COMMAND                  CREATED             STATUS                    PORTS                                                    NAMES
7e9344a2157f        docksal/ssh-agent:1.2       "docker-entrypoint.s…"   20 minutes ago      Up 20 minutes (healthy)                                                            docksal-ssh-agent
364acab01d5e        docksal/dns:1.1             "docker-entrypoint.s…"   20 minutes ago      Up 20 minutes (healthy)   192.168.64.100:53->53/udp                                docksal-dns
a22f8d0b7d94        docksal/vhost-proxy:1.5     "docker-entrypoint.s…"   21 minutes ago      Up 20 minutes (healthy)   192.168.64.100:80->80/tcp, 192.168.64.100:443->443/tcp   docksal-vhost-proxy
0179e7de9755        docksal/web:2.1-apache2.4   "httpd-foreground"       5 hours ago         Up 19 minutes             80/tcp, 443/tcp                                          flyontario_web_1
d3b0be73bda4        docksal/db:1.1-mysql-5.6    "/entrypoint.sh mysq…"   5 hours ago         Up 19 minutes             0.0.0.0:32768->3306/tcp                                  flyontario_db_1
458dfeb533db        docksal/cli:2.5-php5.6      "/opt/startup.sh sup…"   5 hours ago         Up 19 minutes (healthy)   22/tcp, 3000/tcp, 9000/tcp                               flyontario_cli_1
f62fbc59299a        mailhog/mailhog             "MailHog"                7 days ago          Up 19 minutes             80/tcp, 1025/tcp, 8025/tcp                               flyontario_mail_1

███  DOCKER: NETWORKS
NETWORK ID          NAME                 DRIVER              SCOPE
ed32e971ebd1        _default             bridge              local
62671fde59c6        bridge               bridge              local
9c085ea8930f        flyontario_default   bridge              local
6520a37a1d78        host                 host                local
6c372123048b        none                 null                local

███  DOCKER: IMAGES
REPOSITORY            TAG                 IMAGE ID            CREATED             SIZE
nginx                 latest              27a188018e18        9 days ago          109MB
busybox               latest              af2f74c517aa        3 weeks ago         1.2MB
docksal/ssh-agent     1.2                 13bd896a38e9        3 weeks ago         14.3MB
docksal/ssh-agent     latest              53c481f7d019        3 weeks ago         14.3MB
docksal/dns           1.1                 0bed817503b8        3 weeks ago         5.92MB
docksal/vhost-proxy   1.5                 5060de4906ef        3 weeks ago         151MB
docksal/web           latest              3d52def5cc85        2 months ago        91.7MB
docksal/vhost-proxy   latest              333f74b7728f        5 months ago        151MB
mailhog/mailhog       latest              e00a21e210f9        6 months ago        19.2MB
docksal/cli           2.5-php5.6          c90996c98712        6 months ago        800MB
docksal/empty         latest              6c6084ed97e5        15 months ago       3.99MB
docksal/cli           1.3-php5            80248f5ea845        19 months ago       814MB
docksal/web           2.1-apache2.4       32c1d32eba6d        20 months ago       87.5MB
docksal/web           2.1-apache2.2       f64644fd6d46        20 months ago       81MB
docksal/db            1.1-mysql-5.6       6feb79af9597        21 months ago       298MB
docksal/db            1.1-mysql-5.5       aa748dc21dd4        21 months ago       256MB

███  HDD Usage
Filesystem                 Size  Used Avail Use% Mounted on
udev                       5,9G     0  5,9G   0% /dev
tmpfs                      1,2G  9,8M  1,2G   1% /run
/dev/mapper/mint--vg-root  909G  688G  176G  80% /
tmpfs                      5,9G  102M  5,8G   2% /dev/shm
tmpfs                      5,0M  4,0K  5,0M   1% /run/lock
tmpfs                      5,9G     0  5,9G   0% /sys/fs/cgroup
/dev/loop1                 261M  261M     0 100% /snap/phpstorm/87
/dev/loop0                 264M  264M     0 100% /snap/phpstorm/95
/dev/loop3                  91M   91M     0 100% /snap/core/6405
/dev/loop5                  92M   92M     0 100% /snap/core/6531
/dev/loop2                 264M  264M     0 100% /snap/phpstorm/90
/dev/loop4                  90M   90M     0 100% /snap/core/6673
/dev/sda1                  472M  200M  248M  45% /boot
cgmfs                      100K     0  100K   0% /run/cgmanager/fs
tmpfs                      1,2G   36K  1,2G   1% /run/user/1000
/home/wfraga/.Private      909G  688G  176G  80% /home/wfraga

███  RAM Usage
              total        used        free      shared  buff/cache   available
Mem:       12242964     3435948      193104      316532     8613912     7989312
Swap:       8334844          12     8334832

███  Running containers stats
CONTAINER ID        NAME                  CPU %               MEM USAGE / LIMIT     MEM %               NET I/O             BLOCK I/O           PIDS
7e9344a2157f        docksal-ssh-agent     0.00%               4.082MiB / 11.68GiB   0.03%               4.9kB / 0B          21.3MB / 0B         2
364acab01d5e        docksal-dns           0.00%               2.969MiB / 11.68GiB   0.02%               50kB / 59.4kB       6.9MB / 0B          1
a22f8d0b7d94        docksal-vhost-proxy   0.42%               25.13MiB / 11.68GiB   0.21%               24.6kB / 4.59kB     225MB / 32.8kB      15
0179e7de9755        flyontario_web_1      0.18%               6.773MiB / 11.68GiB   0.06%               5.26kB / 0B         32MB / 0B           82
d3b0be73bda4        flyontario_db_1       0.12%               495.8MiB / 11.68GiB   4.15%               5.7kB / 0B          131MB / 106kB       21
458dfeb533db        flyontario_cli_1      0.05%               38.73MiB / 11.68GiB   0.32%               5.7kB / 0B          270MB / 16.4kB      6
f62fbc59299a        flyontario_mail_1     0.00%               3.012MiB / 11.68GiB   0.03%               9.55kB / 0B         36.7MB / 0B         6
███  Docksal Proxy status
nginx: the configuration file /usr/local/openresty/nginx/conf/nginx.conf syntax is ok
nginx: configuration file /usr/local/openresty/nginx/conf/nginx.conf test is successful
  ```
</details>

<br>

**Output of `fin config output`:**

<details>
  <summary>fin config output</summary>

  ```
---------------------
COMPOSE_PROJECT_NAME_SAFE: flyontario
COMPOSE_FILE:
/home/wfraga/.docksal/stacks/volumes-bind.yml
/home/wfraga/.docksal/stacks/stack-default.yml
/var/www/mmda/flyontario/.docksal/docksal.yml
ENV_FILE:
/var/www/mmda/flyontario/.docksal/docksal.env

PROJECT_ROOT: /var/www/mmda/flyontario
DOCROOT: docroot
VIRTUAL_HOST: flyontario.docksal
VIRTUAL_HOST_ALIASES: *.flyontario.docksal
IP: 192.168.64.100
MYSQL: 192.168.64.100:32768

Docker Compose configuration
---------------------
services:
  cli:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      BLACKFIRE_CLIENT_ID: null
      BLACKFIRE_CLIENT_TOKEN: null
      DOCROOT: docroot
      DRUSH_ALLOW_XDEBUG: '0'
      GIT_USER_EMAIL: weydsonfraga3000@gmail.com
      GIT_USER_NAME: wfraga
      HOST_GID: '1000'
      HOST_UID: '1000'
      PHP_IDE_CONFIG: serverName=flyontario.docksal
      SECRET_ACAPI_EMAIL: null
      SECRET_ACAPI_KEY: null
      SECRET_PLATFORMSH_CLI_TOKEN: null
      SECRET_SSH_PRIVATE_KEY: null
      SECRET_TERMINUS_TOKEN: null
      VIRTUAL_HOST: flyontario.docksal
      XDEBUG_CONFIG: remote_connect_back=0 remote_host=192.168.64.1
      XDEBUG_ENABLED: '0'
    hostname: cli
    image: docksal/cli:2.5-php5.6
    volumes:
    - docksal_ssh_agent:/.ssh-agent:ro
    - cli_home:/home/docker:rw
    - project_root:/var/www:rw,nocopy
  db:
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      MYSQL_ALLOW_EMPTY_PASSWORD: null
      MYSQL_DATABASE: default
      MYSQL_INITDB_SKIP_TZINFO: null
      MYSQL_ONETIME_PASSWORD: null
      MYSQL_PASSWORD: user
      MYSQL_RANDOM_ROOT_PASSWORD: null
      MYSQL_ROOT_PASSWORD: root
      MYSQL_USER: user
    hostname: db
    image: docksal/db:1.1-mysql-5.6
    ports:
    - 3306/tcp
    volumes:
    - db_data:/var/lib/mysql:rw
    - project_root:/var/www:ro,nocopy
  mail:
    environment:
      MH_API_BIND_ADDR: 0.0.0.0:80
      MH_UI_BIND_ADDR: 0.0.0.0:80
    expose:
    - '80'
    hostname: mail
    image: mailhog/mailhog
    labels:
      io.docksal.virtual-host: webmail.flyontario.docksal
    user: root
  web:
    depends_on:
      cli:
        condition: service_started
    dns:
    - 192.168.64.100
    - 8.8.8.8
    environment:
      APACHE_BASIC_AUTH_PASS: null
      APACHE_BASIC_AUTH_USER: null
      APACHE_DOCUMENTROOT: /var/www/docroot
      APACHE_FCGI_HOST_PORT: cli:9000
    hostname: web
    image: docksal/web:2.1-apache2.4
    labels:
      io.docksal.cert-name: none
      io.docksal.permanent: "false"
      io.docksal.project-root: /var/www/mmda/flyontario
      io.docksal.virtual-host: flyontario.docksal,*.flyontario.docksal,flyontario.docksal.*
    volumes:
    - project_root:/var/www:ro,nocopy
version: '2.1'
volumes:
  cli_home: {}
  db_data: {}
  docksal_ssh_agent:
    external: true
    name: docksal_ssh_agent
  project_root:
    driver: local
    driver_opts:
      device: /var/www/mmda/flyontario
      o: bind
      type: none

---------------------

  ```
</details>

**Comment by achekulaev** [comment]
So what about your CPU usage?

**Comment by achekulaev** [comment]
@wfraga could you advice on the CPU spikes? If you still have them after the reinstall?

**Comment by lmakarov** [comment]
This has been addressed in docksal/service-vhost-proxy#48 and released as a hotfix in docksal/vhost-proxy [v1.5.1](https://github.com/docksal/service-vhost-proxy/releases/tag/v1.5.1).

Run `fin update` to get the fix.

Please use docksal/service-vhost-proxy#44 as the main issue for any further followups.

**Comment by wfraga** [comment]
I realized that I forgot to thank the help.

Thank you @achekulaev and @lmakarov for all help :)
